sdk/databox/Microsoft.Azure.Management.DataBox/src/Microsoft.Azure.Management.DataBox.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.DataBox</PackageId>
 		<Description>Microsoft Azure Management DataBox library</Description>
 		<AssemblyName>Microsoft.Azure.Management.DataBox</AssemblyName>
-		<Version>1.0.1</Version>
+		<Version>1.2.0</Version>
```

Comment:
Why is `1.1.0` skipped?

Reply:
Since we didn't refresh our SDK for an year now. We want to bump the version to 1.2.0

Reply:
Time shouldn't be the reason to skip a minor version number.